src/core/components/SurveyNotice/index.js

```diff
@@ -26,40 +33,52 @@ type InternalProps = {|
   _config: typeof config,
   _cookie: typeof cookie,
   _supportedLangs: Array<string>,
+  _tracking: typeof tracking,
   dispatch: DispatchFunc,
   i18n: I18nType,
   siteLang: string,
   wasDismissed: boolean,
 |};
 
-export const SurveyNoticeBase = ({
-  _config = config,
-  _cookie = cookie,
-  _supportedLangs = [
-    'de',
-    'en-US',
-    'es',
-    'fr',
-    'ja',
-    'pl',
-    'pt-BR',
-    'ru',
-    'zh-CN',
-    'zh-TW',
-  ],
-  ...props
-}: InternalProps) => {
-  const { dispatch, i18n, location, siteLang, wasDismissed } = props;
-
-  if (
-    wasDismissed ||
-    !_supportedLangs.includes(siteLang) ||
-    !_config.get('enableExperienceSurvey')
-  ) {
-    return null;
+export class SurveyNoticeBase extends React.Component<InternalProps> {
+  static defaultProps = {
+    _config: config,
+    _cookie: cookie,
+    _supportedLangs: [
+      'de',
+      'en-US',
+      'es',
+      'fr',
+      'ja',
+      'pl',
+      'pt-BR',
+      'ru',
+      'zh-CN',
+      'zh-TW',
```

Comment:
Would it make sense to have this array in `config/`?

Reply:
It would add some indirection and I don't think we'll need it. The survey will be disabled soon.